packages/web3torrent/src/library/web3torrent-lib.ts

```diff
@@ -256,14 +280,20 @@ export default class WebTorrentPaidStreamingClient extends WebTorrent {
   protected async transferFunds(wire: PaidStreamingWire) {
     const channelId = wire.paidStreamingExtension.peerChannelId;
 
-    await this.paymentChannelClient.makePayment(
-      channelId,
-      WEI_PER_BYTE.mul(BUFFER_REFILL_RATE).toString()
-    );
-    const newSeederBalance = bigNumberify(
-      this.paymentChannelClient.channelCache[channelId].beneficiaryBalance
-    );
-    log(`payment made for channel ${channelId}, newSeederBalance: ${newSeederBalance}`);
+    if (
+      this.paymentChannelClient.channelCache[channelId] &&
+      this.paymentChannelClient.channelCache[channelId].status === 'running' &&
+      bigNumberify(this.paymentChannelClient.channelCache[channelId].turnNum).gt(3)
+    ) {
```

Comment:
This may not be the correct fix. It may allow leecher may skip the first payment. See https://github.com/statechannels/monorepo/pull/1163#discussion_r390678890

Reply:
The `turnNum = 3` state is not really a payment, it is a setup state that confirms the initial balance of the channel. It is handled by the `joinChannel` call from the leecher.

This condition is there to stop the leecher sending a payment (e.g. `turnNum = 5` state) before they have received the "special" `turnNum = 4` state from the seeder. I'm going to call the `turnNum = 4` state the "spacer state" because it doesn't do anything useful save for increasing the `turnNum` and allowing the leecher to take the next step.